Assets/Scripts/Framework/Gameplay/Values/SessionValuesShortcut.cs

```diff
@@ -0,0 +1,139 @@
+using ProefExamen.Framework.Utils;
+using UnityEngine;
+
+using ProefExamen.Framework.Gameplay.MapData;
+
+namespace ProefExamen.Framework.Gameplay.Values
+{
+    /// <summary>
+    /// A class responsible for setting the default variables of the static class SessionValues. 
+    /// Is also used to view values of SessionValues realtime which is usefull for debuggin.
+    /// </summary>
+    public class SessionValuesShortcut : AbstractSingleton<SessionValuesShortcut>
```

Comment:
This script is only used in editor right? If this is true, I would change the name to something like `SessionValuesEditor`